### ISSUE TITLE
Can't authorize cypress-io to use github

### ISSUE THREAD

**Issue by btalexraescu** [cypress-io/cypress#1251]
### Current behavior:
`Authorize cypress-io` button is disabled.

### Desired behavior:
The button should not be disabled.

### How to reproduce:
Fresh install of cypress through `npm install cypress --save-dev`. Start cypres `npx cypress open
`, login with git account. After login, I'm redirected to authorize screen. The`Authoryze button` is disabled, and can't finish log in process.

### Additional Info (images, stack traces, etc)
Also I get a banner saying that Github no longer supports my web browser.
I think that the issue might be related to another electron issue:

https://github.com/electron/electron/issues/8838

<img width="995" alt="screen shot 2018-02-02 at 14 06 56" src="https://user-images.githubusercontent.com/25976004/35733018-59e9c9aa-0824-11e8-832e-6d1956cda294.png">

- Operating System: Mac OS High Sierra 10.13.3
- Cypress Version: 1.4.1
- Browser Version:Chrome 63

**Comment by jennifer-shehane** [comment]
Electron fix: https://github.com/manosim/gitify/commit/a5b326f1f4e93b10e43d7985bdb4cc55848103c8

**Comment by brian-mann** [comment]
We already do that - so it's not the root issue.

**Comment by olivvv** [comment]
I have the same issue with the desktop version on win10

**Comment by saadaouad** [comment]
Me too i get the same issue!, i think something is wrong with auth0 config.

![image](https://user-images.githubusercontent.com/17694380/35894126-2fdade46-0ba9-11e8-93e1-b529fd9b0ebf.png)

**Comment by jennifer-shehane** [comment]
I've searched our auth0 logs and see no indications of errors in this part. 

How the button disabling works:

> The button is always initially disabled, but it is re-enabled automatically by JavaScript running in that page. (This behavior is there to prevent certain types of attacks.) You'll see this if you reload the page in your browser and look closely at the button --you'll see it's disabled at first and then is enabled after a second or so.

I see the note about upgrading the browser, but am unable to replicate the disabled authorize button. I tried:
- initial npm install in new project
- running cypress locally (no errors in console)

Needs more information or investigation.

**Comment by saadaouad** [comment]
FYI @jennifer-shehane, 
- i'm in the latest version of chrome
![image](https://user-images.githubusercontent.com/17694380/35954975-991d30a8-0c84-11e8-84a4-57b98f906d4d.png)

- `npm` already installed in the app
- cypress is running with no error 
```
➜  react-docs git:(reconfig-circleci) ✗ ./node_modules/.bin/cypress open
GET /__/ 200 38.268 ms - -
GET /__cypress/runner/cypress_runner.css 200 61.953 ms - -
GET /__cypress/runner/cypress_runner.js 200 393.827 ms - -
GET /__cypress/static/favicon.ico 200 3.755 ms - -
GET /__cypress/runner/fonts/fontawesome-webfont.woff2?v=4.7.0 200 2.594 ms - 77160
GET /__cypress/iframes/integration/simple_spec.js 200 359.384 ms - 725
GET /__cypress/tests?p=cypress/integration/simple_spec.js-491 200 1175.646 ms - 828
GET /__cypress/tests?p=cypress/support/index.js-911 200 1238.293 ms - -
GET /__/ 200 10.129 ms - -
GET /__cypress/runner/cypress_runner.css 200 74.171 ms - -
GET /__cypress/runner/cypress_runner.js 200 367.728 ms - -
GET /__cypress/runner/fonts/fontawesome-webfont.woff2?v=4.7.0 200 2.328 ms - 77160
GET /__cypress/iframes/integration/simple_spec.js 200 6.847 ms - 726
GET /__cypress/tests?p=cypress/support/index.js-219 200 6.291 ms - -
GET /__cypress/tests?p=cypress/integration/simple_spec.js-767 200 3.147 ms - 828
GET / 200 4.720 ms - -
GET /assets/css/vendor/fira.css 200 394.537 ms - -
GET /assets/css/styles.a093a01f.css 200 422.839 ms - -
GET /assets/js/vendor/bootstrap.min.c5b5b2fa.js 200 429.758 ms - -
GET /assets/js/vendor/highlight.pack.4cbe7783.js 200 430.034 ms - -
GET /assets/js/vendor/jquery-1.12.0.min.cbb11b58.js 200 505.340 ms - -
GET /assets/css/vendor/bootstrap.min.3cdc65ed.css 200 618.418 ms - -
GET /assets/js/scripts.c16223b1.js 200 346.809 ms - -
GET /assets/fonts/woff/FiraSans-Regular.woff 200 684.548 ms - -
GET /assets/fonts/woff/FiraSans-Medium.woff 200 703.485 ms - -
GET /commands/actions 200 394.363 ms - -
GET /assets/css/vendor/fira.css 200 243.433 ms - -
GET /cdn-cgi/scripts/d07b1474/cloudflare-static/email-decode.min.js 200 251.960 ms - -
GET /assets/css/styles.a093a01f.css 200 256.473 ms - -
GET /assets/css/vendor/bootstrap.min.3cdc65ed.css 200 317.440 ms - -
```

**Comment by jennifer-shehane** [comment]
@saadaouad Thank for the info. Your version of Chrome is irrelevant to where this window is being opened. This is opening within Electron's Chromium browser which is bundled with Cypress. We are in the process of updating Electron so perhaps this issue may be resolved with that update.

**Comment by jennifer-shehane** [comment]
The Electron 59 update #1275 does remove the deprecation banner at the top of the window. 
<img width="1112" alt="screen shot 2018-02-08 at 1 40 44 pm" src="https://user-images.githubusercontent.com/1271364/35991652-d839c41c-0cd5-11e8-8a3d-50f1cacb34da.png">

**Comment by saadaouad** [comment]
>@saadaouad Thank for the info. Your version of Chrome is irrelevant to where this window is being opened. This is opening within Electron's Chromium browser which is bundled with Cypress. We are in the process of updating Electron #1275 so perhaps this issue may be resolved with that update.

Thanks @jennifer-shehane i see now +1

**Comment by saadaouad** [comment]
> The Electron 59 update #1275 does remove the deprecation banner at the top of the window.

Cool

**Comment by saadaouad** [comment]
FYI, you need to login with Github in https://dashboard.cypress.io/#/login, then you should authorize Cypress to use Github within Electron 

![image](https://user-images.githubusercontent.com/17694380/36006585-bbe8dfa4-0d35-11e8-86d3-cae5e3e5620a.png)

**Comment by jennifer-shehane** [comment]
@saadaouad Thanks! Yes, I had read that authorizing through another browser then coming back does solve this. So, yes ^ if you're having this issue go to Dashboard and login then try to come back to application for now. @olivvv @btalexraescu

**Comment by jennifer-shehane** [comment]
Closing due to no new activity recorded with this issue.

**Comment by guillaumep** [comment]
Got the same issue while trying to create a new account from the Cypress application on macOS.

The 'Authorize cypress-io' button is disabled and not clickable.

<img width="1010" alt="capture d ecran le 2019-02-12 a 16 15 56" src="https://user-images.githubusercontent.com/1195230/52668438-8bd4a900-2ee1-11e9-8294-2b2d24e7b956.png">

I had to access the URL suggested by saadaouad in a full browser window to make this work.

**Comment by jennifer-shehane** [comment]
We've had 2 people express that they are encountering this issue again. Reopening to investigate.

**Comment by anymost** [comment]
The same issue I encountered. The version of cypress is **3.1.5**

**Comment by anymost** [comment]
I found a solution. firstly you can login in cypress dashboard on your chrome browser, then click the run button in cypress. it will login in successfully.

**Comment by unbracketed** [comment]
I am encountering this issue with current versions installed either via CDN or npm install. 

Cypress Version 3.1.5 (3.1.5)
macOS 10.14.3 (18D109)
npm v6.8.0
<img width="1002" alt="screen shot 2019-02-22 at 1 28 19 pm" src="https://user-images.githubusercontent.com/17127/53272307-c2f74700-36a5-11e9-9b81-82e89807dd24.png">

**Comment by markgoho** [comment]
I've got 2fa enabled on my account and the browser window opens, but doesn't accept my yubikey. It works literally everywhere else except this popup.

**Comment by jennifer-shehane** [comment]
This PR is in progress and should fix this issue. It will be released as part of 3.1.6. https://github.com/cypress-io/cypress/issues/3583
